tests/Feature/FavouriteTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Pet;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class FavouriteTest extends TestCase
```

Comment:
`$pet` and `$user` are created from factory in almost every test, instead of that create one in `setUp` globally and then use it where you need by doing `$this->user` `$this->pet`